src/PostgresWebsockets/Claims.hs

```diff
@@ -12,14 +12,15 @@ import qualified Crypto.JOSE.Types   as JOSE.Types
 import           Crypto.JWT
 import qualified Data.HashMap.Strict as M
 import           Protolude
+import Data.List
 import Data.Time.Clock (UTCTime)
 import Data.String (String, fromString)
 import qualified Data.Aeson as JSON
 import qualified Data.Aeson.Types as JSON
 
 
 type Claims = M.HashMap Text JSON.Value
-type ConnectionInfo = (ByteString, ByteString, Claims)
+type ConnectionInfo = ([ByteString], ByteString, Claims)
```

Comment:
we should add this type to the list of export so we can use in explicit type annotation in `PostgresWebscokets` module